core/validation/pre_transformation_validate.py

```diff
@@ -0,0 +1,131 @@
+                     DataFrames representing each sheet in the submission.
+    :param schema: The schema containing wrong input checks.
+    :raises ValidationError: If wrong input validation fails.
+    """
+    wrong_input_checks = schema["wrong_input_checks"]
+
+    failures = check_values(workbook, wrong_input_checks, WrongInputFailure)
+
+    if failures:
+        raise ValidationError(validation_failures=failures)
+
+
+def conflicting_input_validation(workbook: dict[str, pd.DataFrame], schema: dict):
+    """Performs conflicting input validation based on the provided schema.
+
+    If the input for two fields is valid, but one field cannot be submitted with another
+    (e.g. if a 'place_name' does not have a certain 'fund_type' but the submission contains both)
+    then this will raise an error.
+
+    :param workbook: A dictionary where keys are sheet names and values are pandas
+                     DataFrames representing each sheet in the submission.
+    :param schema: The schema containing conflicting input checks.
+    :raises ValidationError: If conflicting input validation fails.
+    """
+    conflicting_input_checks = list()
+    for sheet, column_1, row_1, column_2, row_2, mapping, type in schema["conflicting_input_checks"]:
+        # use col_2 and row_2 to map the value we expect based on the input to the type of input from col_1 and row_1
+        expected_values = tuple(mapping.get(str(workbook[sheet].iloc[column_2][row_2]).strip()))
+
+        conflicting_input_checks.append(
+            PreTransformationCheck(sheet=sheet, column=column_1, row=row_2, expected_values=expected_values, type=type)
+        )
+
+    failures = check_values(workbook, conflicting_input_checks, WrongInputFailure)
+
+    if failures:
+        raise ValidationError(validation_failures=failures)
+
+
+def check_values(
+    workbook: dict,
+    failure_list: list,
+    failure: WrongInputFailure | UnauthorisedSubmissionFailure,
+) -> list[WrongInputFailure] | list[UnauthorisedSubmissionFailure] | None:
+    """Checks values in the workbook against the expected values and returns failures.
+
+    :param workbook: A dictionary where keys are sheet names and values are pandas
+                     DataFrames representing each sheet in the submission.
+    :param failure_list: A list of tuples specifying checks to be performed.
+    :param failure: Type of failure to be checked, e.g., "WrongInputFailure".
+    :return: A list of validation failures (instances of FailureClass) if any, else None.
+    """
+    failures = []
+
+    for sheet, column, row, expected_values, type in failure_list:
+        if workbook[sheet].iloc[column][row] not in expected_values:
+            failures.append(
+                failure(
+                    value_descriptor=type,
+                    entered_value=workbook[sheet].iloc[column][row],
+                    expected_values=expected_values,
+                )
```

Comment:
Bit nit picky here but.. We currently do `workbook[sheet].iloc[column][row]` twice 

I think it also adds clarity to pull it out as a variable like 

```suggestion
    for sheet, column, row, expected_values, type in failure_list:
        entered_value = workbook[sheet].iloc[column][row]
        if  entered_value  not in expected_values:
            failures.append(
                failure(
                    value_descriptor=type,
                    entered_value=entered_value,
                    expected_values=expected_values,
                )
```